site/content/entry/yuzu-progress-report-jan-2023/index.md

```diff
@@ -0,0 +1,491 @@
+Windows users, make sure to periodically update your drivers, as each release introduces major changes, so it’s critical to keep up.
+Intel is finally caring about their GPU division with the release of Arc.
+And finally, as a side note, Xe graphics are sensitive to the use of FSR. They lose a considerable amount of performance, so we strongly recommend using Scaleforce instead.
+
+Now the good parts.
+Getting a proper dual channel configuration not only doubles the available bandwidth, which is the most critical point of iGPU performance, but also raises the EU count to the proper Iris Xe spec you originally paid for.
+
+Benchmarking shows that under these conditions, Xe graphics manage to surpass Vega iGPUs in yuzu, as long as the Intel Vulkan drivers cooperate.
+It also shows that Xe graphics are more tolerant to SMAA than, for example, NVIDIA Pascal GPUs, being an almost free setting, performance wise.
+This, coupled with the better IPC of Intel CPUs, should provide an edge on budget laptops.
+At least, until AMD remembers that not everyone wants to pay a fortune for an RDNA2 integrated GPU and releases a refreshed low end lineup. I got tired of waiting.
+
+As an extra juicy bonus, unlike Arc, the Intel iGPUs still have their amazing native ASTC texture decoder. 
+This means no delays in ASTC heavy games like `Luigi’s Mansion 3`, `Bayonetta 3`, or `Astral Chain`. The almighty RTX 4090 can’t do that.
+
+A small note regarding the ASTC support of Intel iGPUs.
+Not all texture formats used by Switch games are supported by Intel, some will still need to be specially decoded.
+Since these GPUs only have a single queue, Xe users should disable ASTC texture decoding from yuzu’s graphics settings. 
+Most of the time the Xe GPU will be the bottleneck, and making it constantly switch between decoding textures and rendering the game will incur a performance loss.
+Letting the CPU carry this work will help graphical performance. 
+Finally some good use for those E-Cores.
+
+For those that don’t fear noise or heat, tweaking the sliders in [XTU](https://www.intel.com/content/www/us/en/download/17881/29183/intel-extreme-tuning-utility-intel-xtu.html?) is an easy way to get an extra GHz out of their CPUs while plugged in. 
+This can help double the performance in yuzu.
+
+Unfortunately, Intel’s EoL status regarding Windows drivers is starting to affect compatibility with recent games, so we officially consider Gen 9/9.5/11 Intel GPUs to be out of support on yuzu.
+This covers Gen 6 to 11 CPUs, Skylake to Ice Lake.
+We will provide support and log issues for them only if they run on Linux. 
+Or, well, macOS, once MoltenVK is fully working.
+
+## Future projects
+
+I’m still under a sort of “NDA” with the developers, so I’m not allowed to tell you what’s cooking yet, or else.
+
+The intel branch still managed to pass along a bit of information before going dark, `Project Lime`, a co-development between bunnei, Tobi, german77, and byte[].
```

Comment:
```suggestion
Our spies still managed to pass along a bit of information before going dark, `Project Lime`, a co-development between bunnei, Tobi, german77, and byte[].
```

Changed wording as a large section was just talking about "Intel" and this could confuse readers.